Sources/BraintreePaymentFlow/BTPaymentFlowClient.m

```diff
@@ -31,15 +31,15 @@ @interface BTPaymentFlowDriver () <SFSafariViewControllerDelegate>
 
 @end
 
-NSString * const BTPaymentFlowDriverErrorDomain = @"com.braintreepayments.BTPaymentFlowDriverErrorDomain";
+NSString * const BTPaymentFlowClientErrorDomain = @"com.braintreepayments.BTPaymentFlowClientErrorDomain";
 
-@implementation BTPaymentFlowDriver
+@implementation BTPaymentFlowClient
 
-static BTPaymentFlowDriver *paymentFlowDriver;
+static BTPaymentFlowClient *paymentFlowClient;
```

Comment:
Going along with that notification center comment, we might be able to get rid of these static references in the future.

Just a thought, I'm also in the "if it ain't broke, don't fix it" camp here.

Reply:
Totally agree, I think we kick this can down the road for now and get back to it when we rewrite this module in Swift